src/lib/utils.js

```diff
@@ -91,50 +98,66 @@ const getLoggedClient = async (token) => {
-    if (!fs.existsSync(localConfigPath)) {
+const getDeprecatedLocalConfigPath = () => path.join(process.cwd(), DEPRECATED_LOCAL_CONFIG_NAME);
+
+const getJsonFileContent = (filePath) => {
+    if (!fs.existsSync(filePath)) {
         return;
     }
-    return loadJson.sync(localConfigPath);
+    return loadJson.sync(filePath);
 };
 
+const getLocalConfig = () => getJsonFileContent(getLocalConfigPath());
+
+const getDeprecatedLocalConfig = () => getJsonFileContent(getDeprecatedLocalConfigPath());
+
 const getLocalConfigOrThrow = async () => {
     let localConfig = getLocalConfig();
-    if (!localConfig) {
+    let deprecatedLocalConfig = getDeprecatedLocalConfig();
+    if (!localConfig && !deprecatedLocalConfig) {
         return {};
     }
-    // 27-11-2018: Check if apify.json contains old  deprecated structure. If so, updates it.
-    if (localConfig.version && _.isObject(localConfig.version)) {
+
+    // If apify.json exists migrate it to .actor/actor.json
+    if (!localConfig && deprecatedLocalConfig) {
         const answer = await inquirer.prompt([{
             name: 'isConfirm',
             type: 'confirm',
             // eslint-disable-next-line max-len
-            message: 'The new version of Apify CLI uses a new format of the "apify.json" file. Your file will be automatically updated to the new format.',
+            message: 'The new version of Apify CLI uses the ".actor/actor.json" instead of the "apify.json" file. Your "apify.json" file will be automatically updated to the new format.',
         }]);
         if (answer.isConfirm) {
-            try {
-                localConfig = updateLocalConfigStructure(localConfig);
-                writeJson.sync(getLocalConfigPath(), localConfig);
-                info('apify.json was updated, do not forget to commit the new version of apify.json to Git repository.');
-            } catch (e) {
-                throw new Error('Can not update apify.json structure. '
-                    + 'Follow guide on https://github.com/apify/apify-cli/blob/master/MIGRATIONS.md and update it manually.');
-            }
-        } else {
             throw new Error('Command can not run with old apify.json structure. '
                 + 'Follow guide on https://github.com/apify/apify-cli/blob/master/MIGRATIONS.md and update it manually.');
         }
+        try {
+            // Check if apify.json contains old deprecated structure. If so, updates it.
+            if (_.isObject(deprecatedLocalConfig.version)) {
+                deprecatedLocalConfig = updateLocalConfigStructure(deprecatedLocalConfig);
+            }
+            localConfig = {
+                actorSpecification: 1,
+                environmentVariables: deprecatedLocalConfig.env || undefined,
+                ..._.pick(deprecatedLocalConfig, 'name', 'title', 'version', 'buildTag', 'description'),
+            };
+
+            // TODO: Maybe remove the apify.json? Or leave it up to the user...
```

Comment:
I think we should at least remove `apify.json` to `apify.deprecated.json`. If nothing else, it will at least not trigger another migration when the user calls some command next.